common/app/views/fragments/email/stylesheets/main.scala.html

```diff
@@ -28,7 +28,6 @@
     }
 
     h2 {
-        font-weight: bold;
```

Comment:
Good question, I did check this and I don't personally think so. I thought it would be tricky so set up a conditional style that was only bold when the webfont was available (if you have any thoughts on this let me know) and it didn't seem worth the effort given the following difference:

# before
<img width="581" alt="picture 317" src="https://cloud.githubusercontent.com/assets/5122968/19766632/dc4f6026-9c46-11e6-902a-ff7f672a7a7c.png">

# after
<img width="579" alt="picture 318" src="https://cloud.githubusercontent.com/assets/5122968/19766646/e899751a-9c46-11e6-80e6-76080598702f.png">